package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@cks-systems/manifest-sdk",
-    "version": "0.1.4",
+    "version": "0.1.52",
```

Comment:
52?

Reply:
there were some skips between. before was 1.1.5. and then i figured i should start incrementing a bit more slowly...

Reply:
skips meaning i bumped the version without committing a version update.